service/src/graphql/resolvers.ts

```diff
@@ -0,0 +1,60 @@
+import {User} from "./schema";
+import {ApolloError} from 'apollo-server-errors';
+import {createUser, deleteUser, getUserByUserId, getUsers, updateUser} from "../repository/user_respository";
+import {getCoordinatesForAddress, GetCoordinatesResponse} from "../service/geocode_service";
+
+export const graphqlResolvers = {
+    Query: {
+        getAllUsers: async () : Promise<User[]> => {
+            return await getUsers();
+        },
+        getUserByUserId: async (_: any, {id}: {id: string}) : Promise<User> => {
+            return await getUserByUserId(id);
+        },
+        getGeoCode: async (
+            _: never,
+            {address}: {address: string}
+        ): Promise<GetCoordinatesResponse> => {
+            return await getCoordinatesForAddress(address)
+        },
+    },
+    Mutation: {
+        createUser: async (_: any, {input}: {input: User}) : Promise<User> => {
+            try {
+                return await createUser({
```

Comment:
What about this `await`? Would this method still behave the same way if you removed it?